packages/govuk-frontend/src/govuk/components/task-list/task-list.yaml

```diff
@@ -0,0 +1,250 @@
+params:
+  - name: items
+    type: array
+    required: true
+    description: Array of tasks within the task list.
+    params:
+      - name: title
+        type: object
+        required: true
+        description: Object containing the main title for the task.
+        params:
+          - name: text
+            type: string
+            required: true
+            description: Text to use within the title. If `html` is provided, the `text` argument will be ignored.
+          - name: html
+            type: string
+            required: true
+            description: HTML to use within the title. If `html` is provided, the `text` argument will be ignored.
+          - name: classes
+            type: string
+            required: false
+            description: Classes to add to the title wrapper.
+      - name: hint
+        type: object
+        required: false
+        params:
+          - name: text
+            type: string
+            required: true
+            description: Text to use within the hint. If `html` is provided, the `text` argument will be ignored.
+          - name: html
+            type: string
+            required: true
+            description: HTML to use within the hint. If `html` is provided, the `text` argument will be ignored.
+      - name: status
```

Comment:
`item.status.classes` is missing from the docs

https://github.com/alphagov/govuk-frontend/blob/2bb6d8e154454b84b9bf50d9348a8c4cecded5ab/packages/govuk-frontend/src/govuk/components/task-list/template.njk#L23

Reply:
Added, thanks!